tsconfig.json

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
This just boilerplate from create-react-app, but it basically says to compile to ES5 and know that we are using React/JSX.